lib/pharos/cluster_manager.rb

```diff
@@ -40,33 +40,76 @@ def validate
       addon_manager.validate
     end
 
+    # @return [Array<Pharos::Configuration::Host>]
+    def sorted_master_hosts
+      @sorted_master_hosts ||= config.master_hosts.sort_by { |h| master_sort_score(h) }
+    end
+
+    # @param host [Pharos::Configuration::Host]
+    # @return [Integer]
+    def master_sort_score(host)
```

Comment:
Some commentary why we sort the masters would be nice, I'm not sure if I can 100% understand why...

Reply:
Added some commentary.